docs/proposal.md

```diff
@@ -1,74 +0,0 @@
-# Proposal for a Federated Learning Solution between Contoso and DataProvider 
-
-## Summary
-
-This document outlines a Federated Learning (FL) solution for data sharing between DataProvider and Contoso. We propose a plan that will enable leveraging DataProvider's data for improving Contoso's model accuracy without compromising any private data. The partnership will utilize DataProvider's data to adapt Contoso's pretrained model to a new domain (healthcare, finance...). Additionally, Contoso plans to fine-tune the service model based on the data that the company provides. Both training settings will utilize FL. In this document we suggest an architecture and implementation guidelines. 
-
-## Motivation
-
-Due to privacy regulations and limitations created due to trust boundaries, not all data used for training Machine Learning (ML) models can be gathered in a central location to train a model on all available data (_e.g._, some highly sensitive user data located in the DataProvider embassies). This constraint makes it hard to leverage the data for training models in classic training techniques. 
-
-An ML technique called Federated Learning (FL) does just that: train the various models locally on the data available in the clients (_a.k.a._ the silos, and in our case _embassies_), then combine the different models on a central server (_a.k.a._ the _orchestrator_) without data ever leaving the embassy. 
```

Comment:
same comment, saving this?

Reply:
Added it here https://github.com/Azure-Samples/azure-ml-federated-learning/pull/72/files#diff-0b5ca119d2be595aa307d34512d9679e49186307ef94201e4b3dfa079aa89938R13